core/ArchiveProcessor/Rules.php

```diff
@@ -229,13 +230,19 @@ public static function getTodayArchiveTimeToLiveDefault()
     public static function isArchivingDisabledFor(array $idSites, Segment $segment, $periodLabel)
     {
         if ($periodLabel == 'range') {
-            return false;
+            if (StaticContainer::get('archiving.range.force_on_browser_request') !== false) {
+                return false;
+            } else {
+                Log::verbose("Not forcing archiving for range period.");
```

Comment:
By the way in PSR-3 the log level "verbose" doesn't exist, I aliased it to "debug". So using verbose is the same as using debug.